Source/WebCore/platform/graphics/texmap/TextureMapperLayer.h

```diff
@@ -191,6 +211,7 @@ class WEBCORE_EXPORT TextureMapperLayer : public CanMakeWeakPtr<TextureMapperLay
     WeakPtr<TextureMapperLayer> m_effectTarget;
     TextureMapperBackingStore* m_backingStore { nullptr };
     TextureMapperPlatformLayer* m_contentsLayer { nullptr };
+    TextureMapperFlattenedLayer* m_flattenedLayer { nullptr };
```

Comment:
I'd rather use std::unique_ptr<> here, especially in new code.

Reply:
Fixed